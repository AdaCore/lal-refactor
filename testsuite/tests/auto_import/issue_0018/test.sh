auto_import -P prj.gpr --source src/mylib/mylib-timestampedlogger.ads --line 2 --column 48
