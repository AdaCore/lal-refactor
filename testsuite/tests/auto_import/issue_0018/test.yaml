description:
    This test checks that the tool does not crash if the spec where the import
    needs to be added does not contain a body part.

driver: shell_script
