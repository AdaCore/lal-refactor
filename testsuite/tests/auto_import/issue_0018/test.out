Available Imports:
Import: SomeLib.Logger; Qualifier: 
Import: SomeLib.Logger; Qualifier: SomeLib.Logger

Edits:
mylib-timestampedlogger.ads
1:1-1:1 with SomeLib.Logger;


